tasks/src/feature-extraction/about.md

```diff
@@ -42,3 +120,7 @@ for index, features in enumerate(features_list):
 # Shape of input: How are you? = (6, 768)
 ```
 ## Useful Resources
+
+https://huggingface.co/docs/transformers/v4.24.0/en/main_classes/pipelines#transformers.FeatureExtractionPipeline
```

Comment:
Can you write explanations and put them in markdown links? 
e.g. `[your-explanation-here](https://huggingface.co/docs/transformers/v4.24.0/en/main_classes/pipelines#transformers.FeatureExtractionPipeline
Write)`

Reply:
Can you please address this? 🙂